skyportal/tests/api/test_annotation_services.py

```diff
@@ -1,39 +1,53 @@
+import uuid
 import pytest
 from skyportal.tests import api
 
 
 @pytest.mark.flaky(reruns=2)
-def test_irsa_wise(annotation_token, public_source, public_group):
+def test_irsa_wise(annotation_token, public_source, public_group, upload_data_token):
```

Comment:
I think you may be able to call this without `public_source`. It may also be possible to use `upload_data_token` instead of `annotation_token` (but I'm not sure about that last one).

Reply:
you were right.